### PR TITLE
Add Magic Level recipe

### PR DESCRIPTION
**Pull request by adameverson**
Add Magic Level recipe

### CODE REVIEW

**Comment by vraravam** [comment]
Please separate the 2 recipes into individual PRs.

**Comment by adameverson** [comment]
Adjusted.

File: recipes/magic-level/package.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+  "id": "magic-level",
+  "name": "Magic Level",
+  "version": "1.25.111",
```
Comment (vraravam) [inline recipes/magic-level/package.json:4]: If this is a brand new recipe for Ferdium, we should start the version at `1.0.0`

**Review by vraravam — CHANGES_REQUESTED** [review]

**Comment by vraravam** [comment]
@all-contributors please add @adameverson for code

**Pull request merged**